<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPlayer</class>
 <widget class="QMainWindow" name="MediaPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MediaPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="hzVolume">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbBack">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbPlay">
       <property name="text">
        <string>P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStop">
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbForward">
       <property name="text">
        <string>F</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListView" name="lTunes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>331</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_files">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionAdd_files"/>
    <addaction name="separator"/>
    <addaction name="actionClear_list"/>
    <addaction name="actionSave_list"/>
    <addaction name="actionload_list"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuAdd_files"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_files">
   <property name="text">
    <string>Add files</string>
   </property>
  </action>
  <action name="actionSave_list">
   <property name="text">
    <string>Save list</string>
   </property>
  </action>
  <action name="actionload_list">
   <property name="text">
    <string>load list</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClear_list">
   <property name="text">
    <string>Clear list</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
